_posts/en/newsletters/2024-04-03-newsletter.md

```diff
@@ -0,0 +1,123 @@
+  policy (see [Newsletter #37][news37 confiscation]).  We're unaware of
+  any actual use of such a script, either in the 10 years Bitcoin
+  existed before the original consensus cleanup proposal or the 5 years
+  since, although some types of use would be impossible to detect until
+  a presigned transaction was broadcast.
+
+  To address the concern, Poinsot proposed that the updated consesus
+  rules only apply to transaction outputs created after a particular
+  block height.  Any outputs created earlier than that height would
+  still be spendable under the old rules.
+
+- **Choosing new BIP editors:** Mark "Murch" Erhardt continued the
+  [thread][erhardt bip editors] about adding new BIP editors by
+  proposing everyone express their "their arguments for and against any
+  candidates in this thread until Friday end-of-day (April 5th).  If any
+  candidates find broad support, those candidates could be added as new
+  editors to the repository on the following Monday (April 8th)."
+
+    Discussion was ongoing at the time of writing and we will do our
+    best to report on the results in next week's newsletter.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 26.1rc2][] is a release candidate for a maintenance release
+  of the network's predominant full node implementation.
+
+- [Bitcoin Core 27.0rc1][] is a release candidate for the next major
+  version of the network's predominant full node implementation.
+  There's a brief overview to [suggested testing topics][bcc testing].
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Bitcoin Inquisition][bitcoin inquisition repo], and [BINANAs][binana
+repo]._
+
+*Note: the commits to Bitcoin Core mentioned below apply to its master
+development branch and so those changes will likely not be released
+until about six months after the release of the upcoming version 27.*
+
+- [Bitcoin Core #27307][] begins tracking the txid of transactions in
+  the mempool that conflict with a transaction belonging Bitcoin Core's
+  built-in wallet.  This includes transactions in the mempool that
+  conflict with an ancestor of a wallet transaction.  If conflicting
+  transaction gets confirmed, the wallet transaction cannot be included
+  on that blockchain, so it's very useful to know about conflicts.
+
+- [Bitcoin Core #29242][] Mempool util: Add RBF diagram checks for single chunks against clusters of size 2 FIXME:harding
```

Comment:
If you are not already working on that, I could write something here. I reviewed that PR and am following closely the cluster mempool discussion, otherwise happy to review what you wrote later. Please let me know.

Reply:
@murchandamus that'd be great!

Reply:
Okay, will start right away.